models/demos/lenet/tt/tt_lenet.py

```diff
@@ -0,0 +1,110 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import ttnn
+import torch.nn as nn
+
+
+def conv(device, input_tensor, batch_size, parameters):
+    weight = parameters.weight
+    bias = parameters.bias
+
+    conv_config = ttnn.Conv2dConfig(
+        dtype=ttnn.bfloat16,
+        weights_dtype=ttnn.bfloat16,
+        math_fidelity=ttnn.MathFidelity.LoFi,
+        activation="relu",
+        shard_layout=ttnn.TensorMemoryLayout.HEIGHT_SHARDED,
+        math_approx_mode_enabled=True,
+        fp32_dest_acc_enabled=False,
+        packer_l1_accum_enabled=False,
+        input_channels_alignment=32,
+        transpose_shards=False,
+        reshard_if_not_optimal=True,
+        deallocate_activation=True,
+        reallocate_halo_output=True,
+    )
+    [x, out_height, out_width, weights_device, bias_device] = ttnn.conv2d(
+        input_tensor=input_tensor,
+        weight_tensor=weight,
+        in_channels=input_tensor.shape[3],
+        out_channels=weight.shape[0],
+        device=device,
+        bias_tensor=bias,
+        kernel_size=(5, 5),
+        stride=(1, 1),
+        padding=(0, 0),
+        batch_size=batch_size,
+        input_height=input_tensor.shape[1],
+        input_width=input_tensor.shape[2],
+        conv_config=conv_config,
+        conv_op_cache={},
+        debug=True,
+        groups=1,
+    )
+    return x, out_height, out_width
+
+
+def Lenet(input_tensor, model, batch_size, num_classes, device, parameters, reset_seeds):
```

Comment:
PEP8 specifies that function are `snake_case`. Consider updating this function name.